modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -98,6 +61,19 @@ def get_pdf
 
         private
 
+        def process_claim(auto_claim)
+          ClaimsApi::V2::DisabilityCompensationPdfGenerator.perform_async(
+            auto_claim.id,
+            veteran_middle_initial, # PDF mapper just needs middle initial
+            @file_number # EVSS mapper needs this number
```

Comment:
this would log the file number in our Sidekiq UI. It shows all params sent in. I thought our idea was to create a wrapper or build the vet so remove this requirement.